specification/src/chapter2.adoc

```diff
@@ -329,6 +403,7 @@ a| * Glitching to bypass secure boot
 * Retrieving residual confidential memory after a system reset
 a| * Implement robust power management
 * Implement cryptographic memory protection with at least boot freshness
+* Code hardening (for example, see https://www.riscure.com/publication/practical-steps-evaluate-protect-secure-boot-implementations-embedded-devices/[Riscure white paper])
```

Comment:
For references can include:
NIST SP 800-193 “Platform Firmware Resiliency Guidelines”
NIST SP 800-57 “Recommendation for Key Management”
NIST SP 800-147 “BIOS Protection Guidelines”
NIST SP 800-147B “BIOS Protection Guidelines for Servers”